src/ui/common/src/components/pages/workflow/id/index.tsx

```diff
@@ -482,6 +482,25 @@ const WorkflowPage: React.FC<WorkflowPageProps> = ({
           </Box>
         )}
 
+        {/*Show any workflow-level errors at the top of the workflow details page.*/}
+        {workflow.selectedResult.exec_state.error !== null && (
```

Comment:
Tiny nit: You can just do `workflow.selectedResult.exec_state.error && (` — Typescript treats `null` and `undefined` as `False`.